Gemfile.lock

```diff
@@ -361,6 +390,7 @@ DEPENDENCIES
   rails!
   rake (>= 10.3)
   redcarpet (~> 3.2.3)
+  redis (~> 3.0)
```

Comment:
Since we never solved this problem, what do you think about having `actioncable` be in the default `Gemfile` instead? We probably shouldn't have the `rails` gem have a hard dependency on `redis`